console/src/components/Services/Data/utils.js

```diff
@@ -791,6 +791,18 @@ WHERE
 export const isColTypeString = colType =>
   ['text', 'varchar', 'char', 'bpchar', 'name'].includes(colType);
 
+export const wrapQuotes = (colType, colDefault) => {
+  const checkIfFunctionFormat = isPostgresFunction(colDefault);
+  if (
+    (isColTypeString(colType) && !checkIfFunctionFormat) ||
+    colType === 'json' ||
+    colType === 'jsonb'
+  ) {
+    return `'${colDefault}'`;
+  }
```

Comment:
This would be needed for other data types as well. E.g. all the date/time types. Could you go over all possible data types and handle them as well? It may turn out that we could wrap everything in quotes.